Source/Core/Core/PowerPC/Interpreter/Interpreter_Integer.cpp

```diff
@@ -582,16 +580,15 @@ void Interpreter::mullwx(UGeckoInstruction inst)
 
 void Interpreter::negx(UGeckoInstruction inst)
 {
-  rGPR[inst.RD] = (~rGPR[inst.RA]) + 1;
+  const u32 a = rGPR[inst.RA];
 
-  if (rGPR[inst.RD] == 0x80000000)
-  {
-    if (inst.OE)
-      PanicAlert("OE: negx");
-  }
+  rGPR[inst.RD] = (~a) + 1;
 
   if (inst.Rc)
     Helper_UpdateCR0(rGPR[inst.RD]);
+
+  if (inst.OE)
+    SetXER_OV(a == 0x80000000);
```

Comment:
Should this be more like `if (inst.OE && a == 0x80000000) SetXER_OV(true);` instead?
I don't read/interpret the PPC Manual as "set when it is but clear when it isn't"; but that might just be me.

Reply:
This would effectively be a NOP if this executes with zero (since set would OR zero into the flags, like what SetXER_SO() already does), however I can hoist it out if that makes it clearer

Reply:
I'm fine with either, but if Interpreter is still considered more like a "reference" implementation I'd try to follow the manual more closely there.

Reply:
Hoisted the condition out like how you suggested it.